src/main/scala/com/microsoft/sqlserver/jdbc/spark/utils/BulkCopyUtils.scala

```diff
@@ -223,13 +241,17 @@ object BulkCopyUtils extends Logging {
     * note that both spark and sql should be configured with the same case settings
     * i.e both CaseSensitive or both not CaseSensitive. In scenario where spark case settings is
     * different this check may pass, but there will be an error at write.
+    * @param conn: Connection,
+    * @param options: SQLServerBulkJdbcOptions,
     * @param df: DataFrame,
     * @param rs: ResultSet,
     * @param url: String,
     * @param isCaseSensitive: Boolean
     *
     */
     private[spark] def matchSchemas(
```

Comment:
scenarios to test :  0, 1, 2, 3, all computed cols.

Reply:
I tested 0, 1, 2, 3 computed cols, since the sql table has to have at least one non computed col.

Reply:
Thanks.